web-platform/website/src/components/pages/home/study-card/ToastComponent.tsx

```diff
@@ -32,67 +37,91 @@ export function ToastComponent({
   const [isDismissed, setIsDismissed] = useState<boolean>(false);
 
   return (
-    <Container className={styles.alert}>
-      <Alert className={`${type} `} isOpen={isShown && !isDismissed}>
-        <div className={`d-flex align-items-center justify-content-between `}>
-          <div className="left d-flex">
-            <img
-              width="24px"
-              height="24px"
-              className="warning-icon"
-              src={icon}
-              alt="warning icon"
-            />
+    <Container
+      className={`${styles.alert} ${ToastStyle.productToast} ${
+        isShown ? "" : styles.hide
```

Comment:
see my comment further down; if you move `height: "38px"` from the Container to the Alert, you can bypass this extra logic and remove the need for `styles.hide`

Reply:
I used the wrong property here. I meant to use isDismissed. Thanks for catching.

Reply:
Actually I removed it; it's totally unnecessary lol